Mark update likes (#42)
zarifmahfuz/project-socialdistribution: README.md

| /api/authors/<author_id>/posts/               | Creates a new post for an author [A] | **Retrieves recent posts from an author [A][R]** | - | - |
| /api/authors/<author_id>/posts/<post_id>/                | Update an authors post [A] | **Retrieves an authors post [A][R]** | - | Delete an authors post [A] |
| /api/nodes/             | Add a node [Admin only] | - | - | - |
| /api/authors/<author_id>/posts/<post_id>/likes                | - | **Retrieves a list of likes on an authors post [A][R]** | - | - |

> | /api/authors/<author_id>/liked                | - | **Retrieves a list of likes from an author on public posts [A][R]** | - | - |

Reviewer: ```suggestion
| /api/authors/<author_id>/liked                | - | **Retrieves a list of public things liked by an author [A][R]** | - | - |
```

    },
    {
        "author": {
            "url": "http://127.0.0.1:8000/api/authors/edcfedc2-0c39-40e9-94de-7d234ebf408e/",
            "id": edcfedc2-0c39-40e9-94de-7d234ebf408e,
            "display_name": "UltimateBeast123",
            "profile_image": "",
            "github_handle": "ultimateBeast"
        },
        "post": "http://127.0.0.1:8000/api/authors/edcfedc2-0c39-40e9-94de-7d234ebf408e/posts/9b050b09-97d1-44b7-89ec-d2ed2c23cde1/"
    }    
]
```
#### Possible Status Codes
- `200 OK`
- `401 Unauthorized`
- `404 Not Found`

### Retrieve a list of likes made by an author on public posts
#### Sample Usage
Retreive a list of likes on public posts origninating from author with id 6e3c2a39-8fef-4efb-bb98-0826a7f15f39-
<img width="1131" alt="image" src="https://user-images.githubusercontent.com/77307203/201505044-7aeee91a-1bdc-42f6-84ee-c3c7c515fcde.png">

#### Sample Response
```
[
   {
    "author": {
        "url": "http://127.0.0.1:8000/api/authors/6e3c2a39-8fef-4efb-bb98-0826a7f15f39/",
        "id": "6e3c2a39-8fef-4efb-bb98-0826a7f15f39",
        "display_name": "noob_author",
        "profile_image": "",
        "github_handle": ""
        },

>         post": "http://127.0.0.1:8000/api/authors/6e3c2a39-8fef-4efb-bb98-0826a7f15f39/posts/9b050b09-97d1-44b7-89ec-d2ed2c23cde1/"

Reviewer: ```suggestion
        post: "http://127.0.0.1:8000/api/authors/6e3c2a39-8fef-4efb-bb98-0826a7f15f39/posts/9b050b09-97d1-44b7-89ec-d2ed2c23cde1/"
```
Indentation and type errors here
